<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateWindow</class>
 <widget class="QWidget" name="UpdateWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Software Update</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/ausweisapp.qrc">
    <normaloff>:/images/npa.svg</normaloff>:/images/npa.svg</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="vLayout">
     <item>
      <widget class="QLabel" name="availableLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="text">
        <string>A new version of AusweisApp2 is available!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="downloadLabel">
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="text">
        <string>AusweisApp2 %1 is now available - you have %2. Would you like to download it now?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="locationLabel">
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="text">
        <string>The update file is located at:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="linkLabel">
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="text">
        <string>&lt;a href=&quot;%1&quot;&gt;%1&lt;/a&gt;</string>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextBrowserInteraction</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="title">
        <string>Release Notes:</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QTextEdit" name="releaseNotes">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>80</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::WheelFocus</enum>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="downloadThisUpdateLabel">
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="text">
        <string>Download this update and close current &quot;AusweisApp2&quot;. Install the update and start &quot;AusweisApp2&quot; again.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dlLabel">
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="text">
        <string>When you click &quot;Download update&quot;, this link will be opened in your browser.</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hLayout">
       <item>
        <widget class="QPushButton" name="skipButton">
         <property name="text">
          <string>Skip update</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="reminderButton">
         <property name="text">
          <string>Remind me later</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="downloadButton">
         <property name="text">
          <string>Download update</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
